### ISSUE TITLE
weird int issue -- fixed?

### ISSUE THREAD

**Issue by ofZach** [SFPC/recoded#41]
I think I fixed the weird int slipping to large number bug here -- since min and max of the param seemed fine, it just that val was returning -2,147,483,648 or something similar...  i used a clamp.  just making a note here in case it's helpful. 

https://github.com/SFPC/recoded/blob/master/src/scenes/baseScene.cpp#L276-L277

**Comment by roymacdonald** [comment]
odd. but if it works.....
I'll take a look at it in order to understand the problem.

**Comment by ofZach** [comment]
yeah if you put a scene with an int param and go back and forth you can get the big negative number sometimes just on the first change of it (I put memo scene first and go back and forth with it and heathers).  when I put a break point (if value < -10000 for example) it seems like the param is badly constructed somehow but that min and max are ok....

**Comment by ofZach** [comment]
yeah I can't follow the logic of the parameter changing but there seems to be some kind of bug somwhere in midiParam...
